livre/core/maths/maths.cpp

```diff
@@ -62,8 +62,8 @@ Matrix4f computeModelViewMatrix( const Vector3f& eye, const Vector3f& center )
         up.normalize();
     }
 
-    const Vector3f xAxis = vmml::normalize( up.cross( zAxis ));
-    const Vector3f yAxis = zAxis.cross( xAxis );
+    const Vector3f xAxis = vmml::normalize( vmml::cross( up, zAxis ));
```

Comment:
API consistency. The member func still exists, but is non-const and modifies this. The free func takes to const parameters.